source/common/formatter/stream_info_formatter.cc

```diff
@@ -795,847 +796,1014 @@ class StreamInfoUpstreamSslConnectionInfoFormatterProvider : public StreamInfoFo
-              return FilterStateFormatter::create(format, max_length, false);
-            }}},
-          {"UPSTREAM_FILTER_STATE",
-           {CommandSyntaxChecker::PARAMS_OPTIONAL | CommandSyntaxChecker::LENGTH_ALLOWED,
-            [](const std::string& format, absl::optional<size_t> max_length) {
-              return FilterStateFormatter::create(format, max_length, true);
-            }}},
-          {"DOWNSTREAM_PEER_CERT_V_START",
-           {CommandSyntaxChecker::PARAMS_OPTIONAL,
-            [](const std::string& format, absl::optional<size_t>) {
-              return std::make_unique<DownstreamPeerCertVStartFormatter>(format);
-            }}},
-          {"DOWNSTREAM_PEER_CERT_V_END",
-           {CommandSyntaxChecker::PARAMS_OPTIONAL,
-            [](const std::string& format, absl::optional<size_t>) {
-              return std::make_unique<DownstreamPeerCertVEndFormatter>(format);
-            }}},
-          {"UPSTREAM_PEER_CERT_V_START",
-           {CommandSyntaxChecker::PARAMS_OPTIONAL,
-            [](const std::string& format, absl::optional<size_t>) {
-              return std::make_unique<UpstreamPeerCertVStartFormatter>(format);
-            }}},
-          {"UPSTREAM_PEER_CERT_V_END",
-           {CommandSyntaxChecker::PARAMS_OPTIONAL,
-            [](const std::string& format, absl::optional<size_t>) {
-              return std::make_unique<UpstreamPeerCertVEndFormatter>(format);
-            }}},
-          {"ENVIRONMENT",
-           {CommandSyntaxChecker::PARAMS_REQUIRED | CommandSyntaxChecker::LENGTH_ALLOWED,
-            [](const std::string& key, absl::optional<size_t> max_length) {
-              return std::make_unique<EnvironmentFormatter>(key, max_length);
-            }}},
-          {"UPSTREAM_CONNECTION_POOL_READY_DURATION",
-           {CommandSyntaxChecker::COMMAND_ONLY,
-            [](const std::string&, const absl::optional<size_t>&) {
-              return std::make_unique<StreamInfoDurationFormatterProvider>(
-                  [](const StreamInfo::StreamInfo& stream_info)
-                      -> absl::optional<std::chrono::nanoseconds> {
-                    if (auto upstream_info = stream_info.upstreamInfo();
-                        upstream_info.has_value()) {
-                      if (auto connection_pool_callback_latency =
-                              upstream_info.value()
-                                  .get()
-                                  .upstreamTiming()
-                                  .connectionPoolCallbackLatency();
-                          connection_pool_callback_latency.has_value()) {
-                        return connection_pool_callback_latency;
-                      }
-                    }
-                    return absl::nullopt;
-                  });
-            }}},
-      });
+  CONSTRUCT_ON_FIRST_USE(StreamInfoFormatterProviderLookupTable,
+                         {
+                             {"REQUEST_DURATION",
```

Comment:
Yeah, as @adisuissa mentioned before, most likely your editor changed whitespaces. Please run it through 
`tools/local_fix_format.sh <name-of-the-file>`.

Reply:
Ack - I had a commit locally I forgot to push up, but TIL `tools/local_fix_format.sh` could take a specific file arg instead of just the git diff. Thanks for the tip!

Reply:
To format all changed files I use:

for i in \`git diff --name-only upstream/main\`; do tools/local_fix_format.sh $i; done

Or use specific commit, or HEAD~X instead of upstream/main.

Reply:
@keithmattix I think that formatting is still an issue in this file. I believe that you added 5-6 new formatters, but diff still shows here that every single handler has been moved several spaces to the right.

Can you find in your dev env `clang-format` and check the version? 
Can you also set CLANG_FORMAT env variable to point to clang-format you found?

Reply:
> clang-format version 14.0.0

I'm using the devcontainer provided with VSCode. I ran `./ci/run_envoy_docker.sh './ci/do_ci.sh format'` with `format fix` and with the file name as an argument and still nothing changed

Reply:
I have seen it in the past that what is considered a proper formatting changed between clang-format versions. Let me see in my dev env how it behaves.

Reply:
@keithmattix I think that github is not displaying your latest changes. I cloned your branch and it shows different (correct formatting). Make a small change and push it again. Please avoid doing force-push as it messes up code reviews.

Reply:
@cpakulski Maybe try checking out again? I merged from main and pushed and I'm still seeing the GitHub diff

Reply:
Yeah, I see that it still shows up a huge diff. Can you verify that your latest version of source/common/formatter/stream_info_formatter.cc is indeed different than what GH displays? Based on my clone of your repo, the file was properly formatted but GH displayed wrong thing. If this is still the case, I would suggest closing this PR, creating a new branch in your local dev env, moving the files you changed/added and opening new PR.

Reply:
GitHub is correct according to what I see locally...did you check out the latest commit?

Reply:
Hmm, there is a disconnect somewhere. GH displays for me that in this PR you added +1281 lines and removed -866 lines. Do you think it is correct?

Reply:
I found the issue; clang-format is changing the surrounding code based on the indentation of my changes instead of changing my code to fit the surrounding code (which is the opposite of what I'd expect a formatted to do). I'll play with this and get it worked out; thanks for the help!

Reply:
Yeah, that's unfortunate...
The "easiest" way I'm aware of is reverting a file to a prior git commit-sha, and (possibly) adding the changes that you wanted in the first place.

Reply:
So I'm walking through this manually to try to understand, and something strange is happening: any time I try to add two of these blocks:

```cpp
          {"UPSTREAM_PEER_URI_SAN",
           {CommandSyntaxChecker::COMMAND_ONLY,
            [](const std::string&, absl::optional<size_t>) {
              return std::make_unique<StreamInfoUpstreamSslConnectionInfoFormatterProvider>(
                  [](const Ssl::ConnectionInfo& connection_info) {
                    return absl::StrJoin(connection_info.uriSanPeerCertificate(), ",");
                  });
            }}},
```

Removing one results in no format fix, but if there are 2 added, then it makes the change. Is there some limit on indentation or something in clang-format?